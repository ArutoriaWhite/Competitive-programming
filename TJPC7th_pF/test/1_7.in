71559263 2
36628370 1
52361909 0
