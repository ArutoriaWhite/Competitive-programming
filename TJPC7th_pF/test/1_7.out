36628370 55825724
